spyderlib/widgets/browser.py

```diff
@@ -65,13 +65,24 @@ def find_text(self, text, changed=True,
                   forward=True, case=False, words=False,
                   regexp=False):
         """Find text"""
-        findflag = QWebEnginePage.FindWrapsAroundDocument
+        if not WEBENGINE:
+            findflag = QWebEnginePage.FindWrapsAroundDocument
+        else:
+            findflag = None
+
         if not forward:
-            findflag = findflag | QWebEnginePage.FindBackward
+            if findflag:

```

Comment:
Setting `findflag` to 0 instead of `None` would avoid this `if` switch since the bitwise or would work.